lib/DxilContainer/DxilPipelineStateValidation.cpp

```diff
@@ -360,8 +360,12 @@ void PSVResourceBindInfo1::Print(raw_ostream &OS) const {
 }
 
 void PSVSignatureElement::Print(raw_ostream &OS) const {
+  Print(OS, GetSemanticName());
+}
+
+void PSVSignatureElement::Print(raw_ostream &OS, const char *Name) const {
```

Comment:
I'm trying to figure out the motivation for this change?  Is there some place now where we want a name other than what `GetSemanticName()` returns to be displayed?

Reply:
It is for validation to print Signature Element where SemanticName is not in StringTable.

Reply:
I guess I don't have enough context to understand what that means.  So sometimes PSVSignatureElement doesn't contain the data it needs?

Reply:
It is for emit error message on a PSVSignatureElement generated from DxilSiginatureElement which doesn't have SemanticName setup. The SemanticName will be get from the DxilSiginatureElement.

Reply:
This may be a case of PR being sliced too far away from the code that makes it possible to understand the change.